src/controllers/package.js

```diff
@@ -8,12 +13,47 @@ export const updatePackagesController = (id, price) => {
     return updatePackagesModel(id, price)
 }
 
+export const insertPackageController = async (id, packageDetails) => {
+    const availiable = await getAvailiableSession()
+    if (availiable > 0) {
+        console.log(packageDetails, availiable)
+        await insertPackageModel(id, packageDetails, availiable)
+        return true;
+    }
+    return false;
+}
+
+export const displayRequestPackageController = async () => {
+    return await getRequestpackage()
+}
+
 export const verifyPackagePaymentController = async (id) => {
     const traineeId = await getTraineeIdOfPackage(id)
     if (traineeId) {
-        await activatePackage(traineeId)
+        await activatePackage(id)
         return true;
     }
     return false;
+}
+
+
+export const addScheduleController = async (id, scheduleDetails) => {
+    const packageDetails = await getRemainingSession(id)
+    const checkVehicle = await checkVehicleActive(scheduleDetails.vehicleNameId)
+    const checkUsage = await checkVehicleUsage(scheduleDetails)
+    if (packageDetails > 0 && checkVehicle && checkUsage) {
```

Comment:
use the above variable values to return specific error codes, and use those error codes in handler to return error messages/codes.